machine.go

```diff
@@ -452,6 +457,20 @@ func (m *Machine) SetScratch(scratchsize int64, path string) {
 	}
 }
 
+// SetSwap sets the size and location of on-disk swap space to allocate
```

Comment:
it'd be great to add this to the unit tests

Reply:
I'm thinking of adding `TestSwap` modelled after `TestMemory()`. Namely: set the swap, parse `/proc/swaps` and error out.
If you'd like anything else please let me know.

Reply:
sounds great to me.